[12.0][MIG] account_global_discount: Migration to 12.0

manuelcalerosolis opened the pull request:
This is the migration of module account_global_discount

chienandalu committed:
[ADD][11.0] account_global_discount: New Module

oca-travis committed:
[UPD] Update account_global_discount.pot

OCA-git-bot committed:
[UPD] README.rst

pedrobaeza committed:
[FIX] account_global_discount

Module title incorrect

OCA-git-bot committed:
[UPD] README.rst

chienandalu committed:
[IMP] account_global_discount: Discounts in invoice report

Give discounts info in the invoice report.

OCA-git-bot committed:
account_global_discount 11.0.1.1.0

sergiocorato committed:
[IMP] set name and uom from moves instead of product

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-11.0/account-invoicing-11.0-account_global_discount
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-11-0/account-invoicing-11-0-account_global_discount/

manuelcalerosolis committed:
[MIG] account_global_discount: Migration to 12.0

chienandalu commented on account_global_discount/__manifest__.py:
```diff
@@ -1,8 +1,9 @@
 {
     'name': 'Account Global Discount',
-    'version': '11.0.1.1.0',
+    'version': '12.0.1.1.0',
```
```suggestion
    'version': '12.0.1.0.0',
```

chienandalu commented on account_global_discount/__manifest__.py:
```diff
@@ -1,8 +1,9 @@
 # Copyright 2019 Tecnativa S.L. - David Vidal
+# Copyright 2020 Xtendoo - Manuel Calero
```
Please, just as contributor
```suggestion
```

chienandalu reviewed (changes requested):
Looks like runbot has some warnings

manuelcalerosolis commented:
@pedrobaeza milestone must be 12.0

sergiocorato committed:
[12.0][FIX] too generic xpath  expr

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

fcolus committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (46 of 46 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-stock_picking_invoicing
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-stock_picking_invoicing/pt_BR/

alexis-via committed:
[FIX] account_payment_term_extension: bad use of precision_digits vs precision_rounding

OCA-git-bot committed:
Merge PR #723 into 12.0

Signed-off-by AaronHForgeFlow

OCA-git-bot committed:
account_invoice_refund_reason 12.0.1.0.1

OCA-git-bot committed:
Merge PR #708 into 12.0

Signed-off-by kittiu

oca-travis committed:
[UPD] Update stock_picking_invoicing.pot

OCA-git-bot committed:
stock_picking_invoicing 12.0.2.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-12.0/account-invoicing-12.0-stock_picking_invoicing
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-stock_picking_invoicing/

oca-travis committed:
[UPD] Update stock_picking_invoicing.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-12.0/account-invoicing-12.0-stock_picking_invoicing
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-stock_picking_invoicing/

marylla committed:
Translated using Weblate (German)

Currently translated at 62.5% (5 of 8 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_change_currency
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_change_currency/de/

marylla committed:
Added translation using Weblate (German)

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (15 of 15 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_date_due
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_date_due/de/

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Translated using Weblate (German)

Currently translated at 4.8% (1 of 21 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_refund_reason
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_refund_reason/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 2.6% (1 of 38 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_debitnote
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_debitnote/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 1.0% (1 of 103 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_billing
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_billing/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 3.1% (1 of 32 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_reimbursable
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_reimbursable/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (1 of 1 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_anglo_saxon_no_cogs_deferral
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_anglo_saxon_no_cogs_deferral/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 50.0% (1 of 2 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_repair_link
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_repair_link/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 50.0% (2 of 4 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_blocking
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_blocking/de/

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

stevech091 committed:
Translated using Weblate (Italian)

Currently translated at 28.6% (8 of 28 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_payment_term_extension
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_payment_term_extension/it/

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 97.8% (45 of 46 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-stock_picking_invoicing
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-stock_picking_invoicing/es/

OCA-git-bot committed:
Merge PR #726 into 12.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_payment_term_extension 12.0.1.2.1

chienandalu commented on account_global_discount/models/account_invoice.py:
```diff
@@ -0,0 +1,237 @@
+    discount_display = fields.Char(
+        compute='_compute_discount_display',
+        readonly=True,
+        string="Discount",
```
Just change this label to get rid of the runbot warning

pedrobaeza replied:
```suggestion
        string="Displayed Discount",
```

Commit:
Merge commit 'refs/pull/719/head' of https://github.com/OCA/account-invoicing into 12.0

pedrobaeza commented:
Please avoid merge commits in the PR and there are a lot of commits for translation to German from same person that should be squashed.

We are going to need this module in some weeks, so if you want, we can supersed this.

manuelcalerosolis commented:
ok delete it without problems.

pedrobaeza commented:
Supersed by #748
